content/technology/solutions/altinn-apps/app/apptemplates/asp.net/customizing.md

```diff
@@ -8,7 +8,7 @@ tags: [altinn-apps]
 ## Adding custom API's
 It is possible to add custom API's to the app. The need to be added as web-api controllers in the app project. 
 
-See development handbook for a step to step guideline for adding custom API's to the application.
+See [here](/app/development/api/expose/) for a step to step guideline for adding custom API's to the application.
```

Comment:
```suggestion
See [development handbook](/app/development/api/expose/) for a step to step guideline for adding custom API's to the application.
```

Adding the link to a more descriptive text makes it easier for those using screen readers.